runelite-client/src/main/java/net/runelite/client/plugins/microbot/GirdyScripts/cannonballsmelter/CannonballSmelterConfig.java

```diff
@@ -0,0 +1,19 @@
+package net.runelite.client.plugins.microbot.GirdyScripts.cannonballsmelter;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup("CannonballSmelter")
+public interface CannonballSmelterConfig extends Config {
+    @ConfigItem(
+            keyName = "guide",
+            name = "How to use",
+            description = "Instructions on how to use this plugin",
+            position = 0
+    )
+    default String GUIDE() {
```

Comment:
Please use @ConfigInformation("")

This tag supports html if you want to create a nice layout for the instructions.

You can remove the GUIDE and replace it with @ConfigInformation.

You can place the @ConfigInformation under @CondfigGroup